src/app.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react' //imports react library(JSX) codes from 'react' folder in node-modules and
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/AndrewIndeche/Learning-Project/pullRequest?prid=9896022)
```suggestion
import React from 'react'; //imports react library(JSX) codes from 'react' folder in node-modules and
```